shex/sio-model/8_Consent.shex

```diff
@@ -0,0 +1,26 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```

Comment:
we have to use new version in the base URI. Can you please change `v010` to `v020`?